sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/client.py

```diff
@@ -395,6 +399,92 @@ def mgmt_request(self, message, **kwargs):
 
 
 class SendClient(AMQPClient):
+    """An AMQP client for sending messages.
+
+    :param hostname: The AMQP endpoint to connect to.
+    :type hostname: str
+    :param target: The target AMQP service endpoint. This can either be the URI as
+     a string or a ~pyamqp.endpoint.Target object.
+    :type target: str, bytes or ~pyamqp.endpoint.Target
+    :param auth: Authentication for the connection. This should be one of the subclasses of
+     pyamqp.authentication.AMQPAuth. Currently this includes:
+        - pyamqp.authentication.SASLAnonymous
+        - pyamqp.authentication.SASLPlain
+        - pyamqp.authentication.SASTokenAuth
+     If no authentication is supplied, SASLAnnoymous will be used by default.
+    :type auth: ~pyamqp.authentication.SASLAnonymous or pyamqp.authentication.SASLPlain
+     or pyamqp.authentication.SASTokenAuth
+    :keyword client_name: The name for the client, also known as the Container ID.
+     If no name is provided, a random GUID will be used.
+    :paramtype client_name: str or bytes
+    :keyword debug: Whether to turn on network trace logs. If `True`, trace logs
+     will be logged at INFO level. Default is `False`.
+    :paramtype debug: bool
+    :keyword auto_complete: Whether to automatically settle message received via callback
```

Comment:
I don't think auto_complete works any more.

Reply:
It doesn't, removing it